classes/class-uabb-attachment.php

```diff
@@ -48,7 +48,7 @@ function uabb_attachment_field_cta( $form_fields, $post ) {
 		 * @param array $attachment array, attachment fields from $_POST form.
 		 * @return array $post array, modified post data.
 		 */
-		function uabb_attachment_field_cta_save( $post, $attachment ) {
+		public function uabb_attachment_field_cta_save( $post, $attachment ) {
```

Comment:
please make sure the scope for these functions which you have made public functions is right and everything works well

Reply:
@akshayurankar48 Verify and resolve your comments